### PR TITLE
Add PIM V3 PAG API documentation

### PR DESCRIPTION
**Pull request by mandardalvi**
Tracked via content review work item 59262 in the Skilling ADO.
Supersedes https://github.com/microsoftgraph/microsoft-graph-docs/pull/19585

### CODE REVIEW

**Review by rkarim-ms — APPROVED** [review]

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @mandardalvi, @ilyalushnikov - 

I've left a few comments to address prior to approval. Please review them.

File: api-reference/beta/api/privilegedaccessgroupassignmentschedulerequest-get.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+Read the properties and relationships of a [privilegedAccessGroupAssignmentScheduleRequest](../resources/privilegedaccessgroupassignmentschedulerequest.md) object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from least to most privileged)|
+|:---|:---|
+|Delegated (work or school account)|PrivilegedAssignmentSchedule.ReadWrite.AzureADGroup|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|PrivilegedAssignmentSchedule.ReadWrite.AzureADGroup|
```
[the remark this answers is not in the document]
Comment (ilyalushnikov) [reply api-reference/beta/api/privilegedaccessgroupassignmentschedulerequest-get.md:24]: For all "xxxScheduleRequest entity" related endpoints in PIM for Groups API, we need ReadWrite permission for GET, LIST, filterbycurrentuser, and between Create and cancel.
Current version is accurate.

File: api-reference/beta/api/privilegedaccessgroup-list-eligibilityschedulerequests.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+Get a list of the [privilegedAccessGroupEligibilityScheduleRequest](../resources/privilegedaccessgroupeligibilityschedulerequest.md) objects and their properties.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from least to most privileged)|
+|:---|:---|
+|Delegated (work or school account)|PrivilegedEligibilitySchedule.ReadWrite.AzureADGroup|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|PrivilegedEligibilitySchedule.ReadWrite.AzureADGroup|
```
[the remark this answers is not in the document]
Comment (ilyalushnikov) [reply api-reference/beta/api/privilegedaccessgroup-list-eligibilityschedulerequests.md:24]: For all "xxxScheduleRequest entity" related endpoints in PIM for Groups API, we need ReadWrite permission for GET, LIST, filterbycurrentuser, and between Create and cancel.
Current version is accurate.

[the remark this answers is not in the document]
Comment (ilyalushnikov) [reply api-reference/beta/api/privilegedaccessgroup-list-eligibilityschedulerequests.md:24]: For all "xxxScheduleRequest entity" related endpoints in PIM for Groups API, we need ReadWrite permission for GET, LIST, filterbycurrentuser, and between Create and cancel.
Current version is accurate.

File: api-reference/beta/api/privilegedaccessgroup-list-assignmentschedulerequests.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Get a list of the [privilegedAccessGroupAssignmentScheduleRequest](../resources/privilegedaccessgroupassignmentschedulerequest.md) objects and their properties.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from least to most privileged)|
+|:---|:---|
+|Delegated (work or school account)|PrivilegedAssignmentSchedule.ReadWrite.AzureADGroup|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|PrivilegedAssignmentSchedule.ReadWrite.AzureADGroup|
```
[the remark this answers is not in the document]
Comment (ilyalushnikov) [reply api-reference/beta/api/privilegedaccessgroup-list-assignmentschedulerequests.md:24]: For all "xxxScheduleRequest entity" related endpoints in PIM for Groups API, we need ReadWrite permission for GET, LIST, filterbycurrentuser, and between Create and cancel.
Current version is accurate.

File: api-reference/beta/api/privilegedaccessgroupeligibilityschedulerequest-get.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+Read the properties and relationships of a [privilegedAccessGroupEligibilityScheduleRequest](../resources/privilegedaccessgroupeligibilityschedulerequest.md) object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from least to most privileged)|
+|:---|:---|
+|Delegated (work or school account)|PrivilegedEligibilitySchedule.ReadWrite.AzureADGroup|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|PrivilegedEligibilitySchedule.ReadWrite.AzureADGroup|
```
[the remark this answers is not in the document]
Comment (ilyalushnikov) [reply api-reference/beta/api/privilegedaccessgroupeligibilityschedulerequest-get.md:24]: correct, Read permission isn't enough here

**Review by FaithOmbongi — APPROVED** [review]
Hi @mandardalvi, @ilyalushnikov - Approved.

Let us sync internally about the casing issue.

Also, let's follow up with some tutorials, how-to articles, and guidance on working with rules. We have guidance for [rules for Azure AD roles](https://learn.microsoft.com/en-us/graph/identity-governance-pim-rules-overview) and I believe PIM for groups also has rules?

**Pull request merged**